### PR TITLE
Error response

### PR DESCRIPTION
**Pull request by marcusmoncayo**
I want to propose a change to the @parsed variable. Currently if there is an api response error we create the message using the to_s method on the hash/array. 

This returns an ugly string i.e. "[\"File upload data is larger than the maximum of 1024KB.\"]"

I changed it to join(',')  which will now return 'File upload data is larger than the maximum of 1024KB.'

If there is an array such as [much, awesome, so api, wow]
it will now return  'much awesome,so api,wow'

we are currently doing some message conversion and translation and instead of regexing this in our app I thought it would be more elegant to do this in the error class. Thoughts?

### CODE REVIEW

File: spec/cb/responses/errors_spec.rb [hunk 1]
```diff
@@ -26,6 +28,10 @@ def self.run_specs
             end.to raise_error ApiResponseError
           end
         end
+
+        it 'assigns correct value to @parse' do
+          expect(Responses::Errors.new(@errors_hash, false).parsed).eql? parsed_error_response
+        end
```
Comment (David-Sharpe) [inline spec/cb/responses/errors_spec.rb]: Should you also have a test that tests the joining of the error strings?

File: lib/cb/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.2'
+  VERSION = '17.0.0'
```
Comment (ausmith) [inline lib/cb/version.rb]: Is this really a breaking change? It does now assume that `errors` is an array, but is that a deviation from the rest?

File: spec/cb/responses/errors_spec.rb (hunk 1 above)
Comment (marcusmoncayo) [inline spec/cb/responses/errors_spec.rb]: This does test that...

File: lib/cb/version.rb (hunk 2 above)
Comment (marcusmoncayo) [inline lib/cb/version.rb]: I made it a breaking change in case anyone else was performing string manipulation as well. I was trying to be safe.

Comment (marcusmoncayo) [inline lib/cb/version.rb]: It is still returning a string. Just one that is missing the array markup. 
before 
"[\"File upload data is larger than the maximum of 1024KB.\"]"

now
"File upload data is larger than the maximum of 1024KB.\"

File: spec/cb/responses/errors_spec.rb (hunk 1 above)
Comment (marcusmoncayo) [inline spec/cb/responses/errors_spec.rb]: @errors = ['much awesome', 'so api', 'wow']
parsed_error message = 'much awesome,so api,wow'

File: lib/cb/version.rb (hunk 2 above)
Comment (ausmith) [inline lib/cb/version.rb]: That'd be my vote, but I'm happy to hear other opinions.

**Comment by marcusmoncayo** [comment]
If I view this as a bug, then it could be a patch. Thoughts?

**Comment by tkling** [comment]
This feels like a minor bump to me. It wasn't really a bug (advertised functionality wasn't broken), and to me it doesn't feel like it's an interface change really (string => string).

**Comment by tkling** [comment]
@marcusmoncayo hey you'll still have to run `bundle update ruby-cb-api` on all of the consuming apps no matter what, right? major/minor/patch changes will all require a `bundle update`

**Comment by helious** [comment]
i'm okay with a patch version. 2nding @tkling

**Comment by NicholasBusby** [comment]
looks like a patch to me
yes your changing the signature, but only of the error response. and you could still eat the result as a string if you desired.

**Comment by ausmith** [comment]
I'm going to call that a quorum on this being < major release and I'm good with a minor version bump vs a patch. Merging.

**Comment by tkling** [comment]
:metal:

**Pull request merged**